dev/storage.h

```diff
@@ -826,6 +865,41 @@ namespace sqlite_orm {
             }
 
           protected:
+            bool calculate_remove_add_columns(std::vector<table_xinfo*>& columnsToAdd,
+                                              std::vector<table_xinfo>& storageTableInfo,
+                                              std::vector<table_xinfo>& dbTableInfo);
+            inline void rename_table(sqlite3* db, const std::string& oldName, const std::string& newName) const {
```

Comment:
no need to `inline` member functions. Please remove `inline` - the code will be shorter with the same functionality

Reply:
done